recipe/meta.yaml

```diff
@@ -22,15 +22,19 @@ requirements:
   build:
     - python
     - setuptools
+    - pip
   run:
     - python
     - numpy >=1.8
     - datreant >=1.0.0
-    - click==6.7
+    - click==6.7       # [py<=36]
+    # unpin for py37
+    - click>=6.7       # [py>=37]   
```

Comment:
@mimischi is this a problem?

Reply:
This is being blocked by https://github.com/bio-phys/MDBenchmark/issues/115. I've had a look at it some time ago, but did not resolved it yet.

Reply:
This is fixed now, bio-phys/MDBenchmark#115 has been resolved.